editor/src/key-listener/tool-bar.ts

```diff
@@ -0,0 +1,32 @@
+import { KeyListener, Action, SModelElement } from 'sprotty';
+import { inject } from 'inversify';
+import { matchesKeystroke } from 'sprotty/lib/utils/keyboard';
+import { GLSP_TYPES } from '@eclipse-glsp/client';
+import { SelectionService } from '@eclipse-glsp/client/lib/features/select/selection-service';
+import { JumpAction } from '../jump/action';
+import { QuickActionUI } from '../quick-action/quick-action-ui';
+
+export class ToolBarKeyListener extends KeyListener {
+  @inject(GLSP_TYPES.SelectionService) protected selectionService: SelectionService;
+  @inject(QuickActionUI) protected quickActionUi: QuickActionUI;
+
+  keyDown(element: SModelElement, event: KeyboardEvent): Action[] {
+    if (matchesKeystroke(event, 'KeyJ') && this.isJumpOutActive()) {
+      return [new JumpAction('')];
+    } else {
+      return [];
+    }
+  }
+
+  private isJumpOutActive(): boolean {
+    return this.selectionService.getModelRoot().id.includes('-') && !this.isJumpIntoActive();
+  }
+
+  private isJumpIntoActive(): boolean {

```

Comment:
why is jumpinto active if your in an embedded process? This sounds like a wrong behaviour...

Reply:
Because it is possible to define a subprocess within a subprocess.

Reply:
Ah is not active. Means the key shortcut is the same. 
Cool 👍